src/subscript/sector2fluxnum/sector2fluxnum.py

```diff
@@ -0,0 +1,251 @@
+import argparse
+import sys
+import os
+import datetime
+from cwrap import open
+from ecl.grid import EclGrid
+from ecl.eclfile import EclFile, FortIO
+from subscript.sector2fluxnum import flux_obj
+from subscript.sector2fluxnum import fluxfile_obj as ffo
+from subscript.sector2fluxnum import datafile_obj as do
+from subscript.sector2fluxnum import completions as comp
+
+
+DESCRIPTION = """
+This is a wrapper script that executes the different
```

Comment:
The statement "This is a wrapper script" does not say much. Wrapper of what? Just state explicitlly what sector2fluxnum does.